### ISSUE TITLE
Variable kern with bracket layers

### ISSUE THREAD

**Issue by punchcutter** [googlefonts/fontmake#564]
Not sure if this needs to be handled in fonttools or elsewhere. We're running fontmake on a Glyphs source with output as one variable font. There are a couple glyphs with design changes along the weight axis and those get swapped out using bracket layers. That all works fine, but the kerning disappears with the swap. Glyphs exports it with the interpolated kerning, but with fontmake we lose the kern. I know it's a different glyph so it could theoretically be a totally different shape with different sidebearings requiring different kerning, but it would be nice to keep the variation as Glyphs does.

**Comment by behdad** [comment]
Yeah that wouldn't work with GSUB/GPOS architecture.  The kerning needs to be duplicated for the substituted glyphs..

**Comment by anthrotype** [comment]
this should be handled in glyphsLib https://github.com/googlefonts/glyphsLib/issues/552
